### PR TITLE
#13 Storage.get()

### PR DESCRIPTION
**Pull request by longtimeago**

### CODE REVIEW

**Comment by longtimeago** [comment]
@karato please, find a reviewer

**Comment by karato** [comment]
Let me find a reviewer for this pull request, thanks for submitting it

**Comment by karato** [comment]
@darkled review it

File: src/main/java/com/nerodesk/Storage.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import java.io.InputStream;
+
+/**
+ * File storage.
+ *
+ * @author Paul Polishchuk (ppol@ua.fm)
+ * @version $Id$
+ * @since 0.1
+ * @todo #13:30min Implement put operation to add file to the storage.
+ * @todo #13:30min Implement delete operation to remove file from the storage.
```
Comment (asaen) [inline src/main/java/com/nerodesk/Storage.java:45]: @longtimeago both puzzles too short, they should be at least 20 words length

**Comment by asaen** [comment]
@longtimeago looks nice, just a comment above

File: src/main/java/com/nerodesk/Storage.java (hunk 1 above)
Comment (longtimeago) [inline src/main/java/com/nerodesk/Storage.java:45]: @darkled @yegor256 do we have this restriction documented somewhere?

Comment (asaen) [inline src/main/java/com/nerodesk/Storage.java:45]: @longtimeago it should be documented, unfortunately I cannot find the proper link (maybe @yegor256 can, I guess it is somewhere within `pdd` project), but anyway it is checked during merging with rultor. When puzzle description is too short one gets the error saying something like this

```
ERROR: "puzzle src/main/java/.../MyClazz.java:10-11 has a very short description of just 10 words while a minimum of 20 is required"
```

**Comment by longtimeago** [comment]
@darkled thanks for the remark. Fixed

File: src/main/java/com/nerodesk/Storage.java (hunk 1 above)
Comment (asaen) [inline src/main/java/com/nerodesk/Storage.java:45]: @longtimeago @yegor256 sorry, my fault, I checked it again and it seems there is no such restriction in `nerodesk` yet.

**Comment by asaen** [comment]
@longtimeago thanks!

**Comment by asaen** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@ggajos we should add this `.pdd` file to the repo root:

```
--source=.
--verbose
--rule min-words:20
--rule min-estimate:15
--rule max-estimate:90
```

**Comment by ggajos** [comment]
@yegor256 please see #23

**Comment by asaen** [comment]
@ggajos could you confirm merge request please?

**Comment by ggajos** [comment]
@darkled @yegor256 Something is wrong with my merge permissions. see #23

**Comment by ggajos** [comment]
@rultor merge

**Comment by asaen** [comment]
@ggajos I see. Thinking in terms of our PDD and issue-oriented model you need to create an issue for this, don't you?

**Comment by ggajos** [comment]
@rultor merge

**Comment by karato** [comment]
@darkled done, I added **20 mins** in payment `52471069`; review comments (c=5) added as a bonus; +20 to your rating, your total score is [+1320](http://www.netbout.com/b/35091?open=rating)

**Comment by karato** [comment]
@rultor deploy now pls

**Pull request merged**